src/main/scala/org/broadinstitute/dsde/rawls/webservice/UserApiService.scala

```diff
@@ -92,6 +92,13 @@ trait UserApiService extends HttpService with PerRequestCreator with UserInfoDir
           UserService.props(userServiceConstructor, userInfo),
           UserService.AdminDisableUser(RawlsUserRef(RawlsUserSubjectId(userSubjectId))))
       }
+    } ~
+    path("user" / Segment / "remove") { userSubjectId =>
```

Comment:
what about doing it as an HTTP DELETE on `user/Segment`? though i suppose there's no reason why this one should be the DELETE over disable